wiki/BBCode/en.md

```diff
@@ -338,6 +340,10 @@ The `[lucky]` tag is an outdated tag that was once used in the osu! forums to li
 
 The *Heading (v2)* tag is an outdated tag that was once used in the osu! forums to format text into a fancier-looking, purple heading with a horizontal line. The tag only worked in the Beatmaps forum, and only appeared after posting (not in preview). It had no button when it was in service, and was denoted by an open and closed bracket (no opening and closing tags).
 
+## References
+
+[^imgur-blocked-ip]: [Tweet by @ppy (2023-06-29)](https://twitter.com/ppy/status/1674439849749913602)
+
```

Comment:
footnotes get rendered at the bottom of the page so this needs to be moved all the way down